pkgs/development/python-modules/phik/default.nix

```diff
@@ -0,0 +1,47 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  isPy3k,
+  nbconvert,
+  pytest-pylint,
+  numba,
+  pandas,
+  matplotlib
+}:
+
+buildPythonPackage rec {
+  pname = "phik";
+  version = "0.9.8";
+
+  src = fetchPypi {
+    inherit pname version;
+    format = "wheel";
+    python = "py3";
+    abi = "none";
+    platform = "any";
+    sha256 = "c398452c5c1eea153905666b289c6a153712cf3d58811fa41e2bbbd27a65d678";
+  };
+
+  propagatedBuildInputs = [
+    nbconvert
```

Comment:
These can probably be made `buildInputs` instead of `propagatedBuildInputs`. Really I don't think they should be needed at all, since they're seem to be just for the tests which aren't being run, but I think there is a mistake in the `setup.py` (I think this line shouldn't exist: https://github.com/KaveIO/PhiK/blob/v0.9.8/setup.py#L47) which makes them necessary. Maybe also add `jupyter_client` and `pytest` as `buildInputs`.

Maybe someone more experienced has opinions on whether the 'mistake' I mentioned should be patched out/fixed upstream, or if it's fine as is?

Reply:
Ah yes, https://github.com/NixOS/nixpkgs/pull/71248#discussion_r335763967 works so put them (`nbconvert` and `pytest-pylint`) in `checkInputs`, not `buildInputs`. I'm still convinced the `setup.py` is mistaken, but it shouldn't matter too much.